PWGHF/D2H/Tasks/taskD0.cxx

```diff
@@ -159,10 +160,12 @@ struct HfTaskD0 {
       if (candidate.isSelD0() >= selectionFlagD0) {
         registry.fill(HIST("hMass"), invMassD0ToPiK(candidate), candidate.pt());
         registry.fill(HIST("hMassFinerBinning"), invMassD0ToPiK(candidate), candidate.pt());
+        registry.fill(HIST("hMassVsPhi"), invMassD0ToPiK(candidate), candidate.pt(), candidate.phi());
```

Comment:
At this point it is worth defining variables for the mass and pT.

Reply:
Sure, done